sdk/servicebus/azservicebus/.gitignore

```diff
@@ -0,0 +1,31 @@
+# Binaries for programs and plugins
+*.exe
+*.dll
+*.so
+*.dylib
```

Comment:
You said this is a verbatim merge, but should we add the gitignore to the main gitignore at the project root?